docs/index.md

```diff
@@ -255,6 +255,40 @@ Please follow these simple steps:
 </html>
 ```
 
+#### Viewing distant objects in location-based mode 
+
+If your location-based AR content is distant from the user (around 1km or
+more), it is recommended to use the new `arjs-webcam-texture` component (introduced in AR.js 3.2.0), which uses a THREE.js texture to stream the camera feed and allows distant content to be viewed. This component is automatically injected if the `videoTexture` parameter of the `arjs` system is set to `true` *and* the `sourceType` is `webcam`. For example (code snippet only):
+
+```html
+    <a-scene
+      vr-mode-ui="enabled: false"
+      embedded
+      arjs="sourceType: webcam; videoTexture: true; debugUIEnabled: false;"
+    >
+```
+
+#### Reducing shaking effects in location-based mode
+
+In location-based mode, 'shaking' effects can occur due to frequent small
+changes in the device's orientation, due to the high sensitivity of the device 
+sensors such as the accelerometer. If using AR.js 3.3.1 or greater this can optinally be reduced using an exponential smoothing technique. *Note that there are currently some occasional display artefacts with this if moving the device quickly or suddenly so please test before you enable it in a finished application; work to resolve these is on-going.*
+
+This is enabled by adding the `arjs-look-controls` component to your `a-camera` with a `smoothingFactor` property. You must also disable A-Frame's default `look-controls`, as `arjs-look-controls` will replace it. For example:
```

Comment:
I would specify here: "This parameters can be used to reduce shaking of **distant** places"

Reply:
The smoothing does also reduce shaking (as opposed to jumping) of nearby augmented content though, so not sure this is necessary.